test/data/devicegraphs/output/s390_dasd_zipl.yml

```diff
@@ -1,30 +1,32 @@
 ---
-- disk:
+- dasd:
     name: "/dev/sda"
     size: 50 GiB
-    partition_table: gpt
+    partition_table: dasd
     partitions:
     - partition:
         size: 200 MiB
+        start: 3 MiB
```

Comment:
I would void adding these kind of extra lines that add no information for a human being. The changes on this file should only be line 2, line 5 (hopefully removing the line instead of changing it) and the swap partition if.